Flaky integration test `test_backup_restore_on_cluster/test.py::test_replicated_database_async`

Test failed in CI with release build: https://s3.amazonaws.com/clickhouse-test-reports/44411/a094af9a49fdd7c7994f31588f721093b69b5163/integration_tests__release__[1/4]/integration_run_parallel0_0.log

For this PR https://github.com/ClickHouse/ClickHouse/pull/44411

```
>               raise AssertionError(
                    "'{}' != '{}'\n{}".format(
                        expectation_tsv,
                        val,
                        "\n".join(expectation_tsv.diff(val, n1="expectation", n2="query")),
                    )
                )
E               AssertionError: 'RESTORED' != 'RESTORE_FAILED	Code: 608. DB::Exception: Got error from node1:9000. DB::Exception: Couldn't restore table mydb.tbl on other node or sync it (elapsed 1.042499s): While creating table mydb.tbl. Stack trace:

0. DB::Exception::Exception(DB::Exception::MessageMasked const&, int, bool) @ 0xd1f0f3a in /usr/bin/clickhouse
1. ? @ 0x9f13df4 in /usr/bin/clickhouse
2. DB::DatabaseReplicated::createTableRestoredFromBackup(std::__1::shared_ptr<DB::IAST> const&, std::__1::shared_ptr<DB::Context>, std::__1::shared_ptr<DB::IRestoreCoordination>, unsigned long) @ 0x121860b3 in /usr/bin/clickhouse
3. DB::RestorerFromBackup::createTable(DB::QualifiedTableName const&) @ 0x11dbbfe8 in /usr/bin/clickhouse
4. DB::RestorerFromBackup::createTables() @ 0x11daf335 in /usr/bin/clickhouse
5. DB::RestorerFromBackup::run(DB::RestorerFromBackup::Mode) @ 0x11da9b6c in /usr/bin/clickhouse
6. DB::BackupsWorker::doRestore(std::__1::shared_ptr<DB::ASTBackupQuery> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, DB::BackupInfo const&, DB::RestoreSettings, std::__1::shared_ptr<DB::IRestoreCoordination>, std::__1::shared_ptr<DB::Context>, bool) @ 0x11d78400 in /usr/bin/clickhouse
7. ? @ 0x11d7c373 in /usr/bin/clickhouse
8. ThreadPoolImpl<ThreadFromGlobalPoolImpl<false>>::worker(std::__1::__list_iterator<ThreadFromGlobalPoolImpl<false>, void*>) @ 0xd2bd74a in /usr/bin/clickhouse
9. void std::__1::__function::__policy_invoker<void ()>::__call_impl<std::__1::__function::__default_alloc_func<ThreadFromGlobalPoolImpl<false>::ThreadFromGlobalPoolImpl<void ThreadPoolImpl<ThreadFromGlobalPoolImpl<false>>::scheduleImpl<void>(std::__1::function<void ()>, long, std::__1::optional<unsigned long>, bool)::\'lambda0\'()>(void&&)::\'lambda\'(), void ()>>(std::__1::__function::__policy_storage const*) @ 0xd2bff77 in /usr/bin/clickhouse
10. ThreadPoolImpl<std::__1::thread>::worker(std::__1::__list_iterator<std::__1::thread, void*>) @ 0xd2b9eaa in /usr/bin/clickhouse
11. ? @ 0xd2bef41 in /usr/bin/clickhouse
12. ? @ 0x7fa4399db609 in ?
13. clone @ 0x7fa439900133 in ?
. (CANNOT_RESTORE_TABLE) (version 22.13.1.242)'
E               @@ -1 +1 @@
E               -RESTORED
E               +RESTORE_FAILED	Code: 608. DB::Exception: Got error from node1:9000. DB::Exception: Couldn\'t restore table mydb.tbl on other node or sync it (elapsed 1.042499s): While creating table mydb.tbl. Stack trace:

0. DB::Exception::Exception(DB::Exception::MessageMasked const&, int, bool) @ 0xd1f0f3a in /usr/bin/clickhouse
1. ? @ 0x9f13df4 in /usr/bin/clickhouse
2. DB::DatabaseReplicated::createTableRestoredFromBackup(std::__1::shared_ptr<DB::IAST> const&, std::__1::shared_ptr<DB::Context>, std::__1::shared_ptr<DB::IRestoreCoordination>, unsigned long) @ 0x121860b3 in /usr/bin/clickhouse
3. DB::RestorerFromBackup::createTable(DB::QualifiedTableName const&) @ 0x11dbbfe8 in /usr/bin/clickhouse
4. DB::RestorerFromBackup::createTables() @ 0x11daf335 in /usr/bin/clickhouse
5. DB::RestorerFromBackup::run(DB::RestorerFromBackup::Mode) @ 0x11da9b6c in /usr/bin/clickhouse
6. DB::BackupsWorker::doRestore(std::__1::shared_ptr<DB::ASTBackupQuery> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, DB::BackupInfo const&, DB::RestoreSettings, std::__1::shared_ptr<DB::IRestoreCoordination>, std::__1::shared_ptr<DB::Context>, bool) @ 0x11d78400 in /usr/bin/clickhouse
7. ? @ 0x11d7c373 in /usr/bin/clickhouse
8. ThreadPoolImpl<ThreadFromGlobalPoolImpl<false>>::worker(std::__1::__list_iterator<ThreadFromGlobalPoolImpl<false>, void*>) @ 0xd2bd74a in /usr/bin/clickhouse
9. void std::__1::__function::__policy_invoker<void ()>::__call_impl<std::__1::__function::__default_alloc_func<ThreadFromGlobalPoolImpl<false>::ThreadFromGlobalPoolImpl<void ThreadPoolImpl<ThreadFromGlobalPoolImpl<false>>::scheduleImpl<void>(std::__1::function<void ()>, long, std::__1::optional<unsigned long>, bool)::\'lambda0\'()>(void&&)::\'lambda\'(), void ()>>(std::__1::__function::__policy_storage const*) @ 0xd2bff77 in /usr/bin/clickhouse
10. ThreadPoolImpl<std::__1::thread>::worker(std::__1::__list_iterator<std::__1::thread, void*>) @ 0xd2b9eaa in /usr/bin/clickhouse
11. ? @ 0xd2bef41 in /usr/bin/clickhouse
12. ? @ 0x7fa4399db609 in ?
13. clone @ 0x7fa439900133 in ?
. (CANNOT_RESTORE_TABLE) (version 22.13.1.242)
```

--- a/tests/integration/test_backup_restore_on_cluster/configs/lesser_timeouts.xml
+++ b/tests/integration/test_backup_restore_on_cluster/configs/lesser_timeouts.xml
@@ -2,6 +2,6 @@
     <backups>
         <on_cluster_first_sync_timeout>1000</on_cluster_first_sync_timeout>
         <consistent_metadata_snapshot_timeout>10000</consistent_metadata_snapshot_timeout>
-        <create_table_timeout>1000</create_table_timeout>
+        <create_table_timeout>3000</create_table_timeout>
     </backups>
 </clickhouse>

